build.gradle

```diff
@@ -46,8 +46,8 @@ test {
   }
 }
 
-task wrapper(type: Wrapper) {
-  gradleVersion = '4.0'
+task {
+  gradleVersion = '6.9'
```

Comment:
As per https://docs.gradle.org/4.8/release-notes.html#overwriting-gradle%E2%80%99s-built-in-tasks.

Do we need this?